Makefile

```diff
@@ -3,6 +3,7 @@
 _latest_version := $(shell scripts/version.sh true)
 _version := $(shell scripts/version.sh)
 export USE_CONTAINER ?= docker
+export REBUILD_CONTAINER_IMAGE ?= true
```

Comment:
If defaults for `USE_CONTAINER` and `REBUILD_CONTAINER_IMAGE` are defined in `docker-run.sh`, does it make sense to remove their declarations from the Makefile?

Reply:
`USE_CONTAINER` is also checked in `scripts/run.sh` so it is best pre-set in the `Makefile`. I agree `REBUILD_CONTAINER_IMAGE` can get pulled from there.